content/blog/scale-kubernetes-cluster-using-hpe-greenlake-terraform-provider.md

```diff
@@ -0,0 +1,244 @@
+---
+title: Scale Kubernetes Clusters using HPE GreenLake Terraform Provider
```

Comment:
Just wanted to say "wonderful!" Thank you so much for taking my suggestions into account.

Reply:
No problem. Thanks for all the suggestions.